src/components/UI/Display/Apps/AppList/CatalogLabel.tsx

```diff
@@ -19,18 +25,44 @@ const CatalogType = styled.span`
   color: #222;
 `;
 
+const RedIcon = styled.i`
+  color: ${({ theme }) => theme.colors.redOld};
+`;
+
 export interface ICatalogLabelProps {
   iconUrl: string;
   catalogName: string;
   isManaged?: boolean;
+  error?: string;
 }
 
+const ErrorIcon: React.FC<{ name: string }> = ({ name }) => {
+  return (
+    <OverlayTrigger
+      overlay={
+        <Tooltip id={`app-catalog-load-error-${name}`}>
+          This app catalog could not be loaded. Apps from this catalog cannot be
+          displayed.
+        </Tooltip>
+      }
+      placement='top'
+    >
+      <RedIcon className='fa fa-warning' />
+    </OverlayTrigger>
+  );
+};
+
+ErrorIcon.propTypes = {
+  name: PropTypes.string.isRequired,
+};
+
 const CatalogLabel: React.FC<ICatalogLabelProps> = (props) => {
   return (
-    <Wrapper {...props}>
+    <Wrapper {...props} className={props.error ? 'hasError' : ''}>
```

Comment:
This can be done easier with a custom prop. If `className` would be left alone, we would be able to style this component from other components

Reply:
Fixed